code-of-conduct.md

```diff
@@ -63,7 +63,7 @@ Most of our community are adults and should know when it's ok to flirt with othe
 
 ## Where this CoC is upheld
 
-This code of conduct especially pertains to our community platforms (IRC: [#opensourcedesign](http://chat.opensourcedesign.net), GitHub: [@opensourcedesign](https://github.com/opensourcedesign), Twitter: [@opensrcdesign](https://twitter.com/opensrcdesign)), Facebook: [@opensrcdesign](https://www.facebook.com/opensrcdesign/) but this also pertains to your personal public communications and views.
+This code of conduct especially pertains to our community platforms (IRC: [#opensourcedesign](http://chat.opensourcedesign.net), GitHub: [@opensourcedesign](https://github.com/opensourcedesign), Twitter: [@opensrcdesign](https://twitter.com/opensrcdesign), Facebook: [@opensrcdesign](https://www.facebook.com/opensrcdesign/), Discourse: [discourse.opensourcedesign.net](https://discourse.opensourcedesign.net)) but this also pertains to your personal public communications and views.
```

Comment:
Removed it. :) (You can simply use the online Github editor for quick edits by the way, very cool.)